src/App.tsx

```diff
@@ -317,13 +318,30 @@ export default function App() {
     }
   };
 
+  const selectedOrgHasNurseries = (): boolean => {
+    if (selectedOrganization && selectedOrganization.facilities) {
+      return selectedOrganization.facilities.some((facility) => {
+        return facility.type === 'Nursery';
+      });
+    } else {
+      return false;
+    }
+  };
+
```

Comment:
I understand this is how we did all other empty states - ideally empty state logic should be within the component for Nurseries. App should only do the routing.